.github/workflows/chromatic.yml

```diff
@@ -12,19 +12,25 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+
+      - run: npm install --global corepack@0.31.0
```

Comment:
This is needed as there's an issue with pnpm signatures on the default corepack version. We also had to fix this in support-service-lambdas previously: guardian/support-service-lambdas#2666.